lms/static/js/student_account/views/AccessView.js

```diff
@@ -172,7 +172,8 @@ var edx = edx || {};
         toggleForm: function( e ) {
             var type = $(e.currentTarget).data('type'),
                 $form = $('#' + type + '-form'),
-                $anchor = $('#' + type + '-anchor');
+                $anchor = $('#' + type + '-anchor'),
+                queryParams = '?' + url('?');
```

Comment:
This looks like it will add the "?" even if there are no querystring urls.  Will this make Michael's analysis in GA harder?  (since now there are two URLs for the same page)?